app/models/post.rb

```diff
@@ -4,4 +4,17 @@ class Post < ActiveRecord::Base
 	# ------------------------------------------------------------------------------------------------------
 	belongs_to :user, counter_cache: true
 
+
+	# VALIDATIONS
+	# ------------------------------------------------------------------------------------------------------
+	validates :title, presence: true
+
+
+  # INSTANCE METHODS
+  # ------------------------------------------------------------------------------------------------------
```

Comment:
Line is too long. [106/100]<br>Incorrect indentation detected (column 2 instead of 1).